tests/integration_tests/test_server.py

```diff
@@ -192,6 +192,30 @@ async def test_host_missing_fields(event_loop, lobby_server, player_service):
     assert msg['map_file_path'] == 'maps/scmp_007.zip'
     assert msg['featured_mod'] == 'faf'
 
+@fast_forward(5)
+async def test_host_game_type(event_loop, lobby_server, player_service):
```

Comment:
Might be able to get away with just using the existing tests and adding the `game_type` field to the assertions. I think there is already one for hosting a game.

Reply:
Ok, i'll take a closer look at the testcases. There should be some already which i can use.